apps/modernization-ui/src/configuration/configuration.ts

```diff
@@ -46,6 +46,7 @@ type Features = {
     search: Search;
     patient: {
         add: {
+            enabled: boolean;
```

Comment:
use the type Toggle = { enabled: boolean } here

Reply:
it looks like there is a mix of both being used here besides this story so not necessary to change